### PR TITLE
change log level to debug

### PR DESCRIPTION
**Pull request by ramaraochavali**
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [X] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [X] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/webhooks/validation/controller/controller.go [hunk 1]
```diff
@@ -320,7 +320,7 @@ func (c *Controller) reconcileRequest(req reconcileRequest) (bool, error) {
 	}
 
 	scope.Infof("Reconcile(enter): %v", req)
-	defer func() { scope.Debugf("Reconcile(exit)") }()
+	defer func() { scope.Infof("Reconcile(exit)") }()
```
Comment (howardjohn) [inline pkg/webhooks/validation/controller/controller.go:323]: nit: seems like both should be `debug`, its pretty noisy?

File: pkg/webhooks/validation/controller/controller.go [hunk 2]
```diff
@@ -329,12 +329,18 @@ func (c *Controller) reconcileRequest(req reconcileRequest) (bool, error) {
...
-	if ready {
+	failurePolicy := kubeApiAdmission.Fail
+	// Only cheeck for fail open if webhooks are not already configured for fail.
+	for i := range configuration.Webhooks {
+		if configuration.Webhooks[i].FailurePolicy != nil && *configuration.Webhooks[i].FailurePolicy == kubeApiAdmission.Ignore {
+			failurePolicy = kubeApiAdmission.Ignore
+			break
+		}
+	}
+	if failurePolicy != kubeApiAdmission.Fail && c.readyForFailClose() {
```
Comment (howardjohn) [inline pkg/webhooks/validation/controller/controller.go:340]: I think current code allows changing from fail back to ignore. Are we intentionally removing that?

What is the motivation for this pr?

Comment (ramaraochavali) [reply pkg/webhooks/validation/controller/controller.go:340]: > I think current code allows changing from fail back to ignore. 

I thought it is only doing one way i.e. ignore -> fail . that is why I changed it like that. Let me take a look again.

**Comment by ramaraochavali** [comment]
@howardjohn can you PTAL when you get chance?

**Review by howardjohn — APPROVED** [review]

**Pull request merged**